builders/tools/precompiles/xc20.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Cross-Chain Assets
+description:  Learn how to access and interact with an ERC-20 interface for cross-chain tokens on Moonbeam, and check how this is done on the Moonbase Alpha TestNet by using the assets precompiled contract.
+---
+
+#  Cross-Chain Assets Precompile
+
+![Cross-Chain Assets Precompiled Contracts Banner](/images/builders/tools/precompiles/xc20/xc20-banner.png)
+
+## Introduction {: #introduction } 
+
+The [Cross-Consensus Message (XCM)](https://wiki.polkadot.network/docs/learn-crosschain) format defines how messages can be sent between blockchains. This format opens the door to transfer messages and assets between Moonbeam and the Relay Chain or other parachains.
```

Comment:
First paragraph should be an introduction on the topic.

"XCM enabling native interoperability between the relay chain and/or parachains in the ecosystem"